tools/hangar/__snapshots__/test_corpus/valid/enums.test.w_compile_tf-aws.md

```diff
@@ -54,9 +76,9 @@ class $Root extends $stdlib.std.Resource {
     super($scope, $id);
     const SomeEnum =
       (function (tmp) {
-        tmp[tmp["ONE"] = 0] = ",ONE";
```

Comment:
I suspect this won't be compatible with how TypeScript/JSII represent enums...

Reply:
I think it should work since this is pretty close to how enums are emitted in TypeScript (see screenshot) but I'll keep my eyes peeled for any breakages, unless you have ideas how to test this
 
<img width="665" alt="Screenshot 2024-03-12 at 1 33 38 PM" src="https://github.com/winglang/wing/assets/5008987/7673ebc3-6a4e-4600-8713-5523baee4c3e">